tests/lib/cmsis_dsp/complexmath/testcase.yaml

```diff
@@ -7,7 +7,8 @@ tests:
       - sam_e70_xplained
       - mps2_an521
       - native_sim
-    tags: cmsis_dsp
+      - native_posix
```

Comment:
no need to add native_posix, native_sim covers it, and we will be removing it soon

Reply:
remove this commit as it became unrelated.